src/jv.c

```diff
@@ -610,14 +596,22 @@ static jv jvp_literal_number_new(const char * literal) {
 
 static double jvp_literal_number_to_double(jv j) {
   assert(JVP_HAS_FLAGS(j, JVP_FLAGS_NUMBER_LITERAL));
+  decContext dblCtx;
+
+  dblCtx.digits = DEC_NUBMER_DOUBLE_PRECISION;
+  dblCtx.emax = 384;
+  dblCtx.emin = -383;
+  dblCtx.round = DEC_ROUND_HALF_EVEN;
+  dblCtx.traps = 0;
+  dblCtx.clamp = 1;
```

Comment:
Let's use `decContextDefault(dblCtx, DEC_INIT_DECIMAL64);` and then just override the digits with a comment, wdyt?

Reply:
Yeap better 👍